### PR TITLE
Remove resource groups support for pl/container

### PR DESCRIPTION
**Pull request by dyozie**
Removing RG support for initial 7.0 release.

### CODE REVIEW

File: gpdb-doc/markdown/analytics/pl_container_using.html.md [hunk 1]
```diff
@@ -5,105 +5,20 @@ title: Using PL/Container
...
 
+You cannot, by default, restrict the number of running PL/Container container instances, nor can you restrict the total amount of memory or CPU resources that they consume. Greenplum Database is unaware of the resources consumed by running PL/Container instances. 
 
-### <a id="plc_notes"></a>Notes 
+>**Caution** In this release of Greenplum 7, you cannot use resource groups to manage and limit the total CPU and memory resources for a PL/Container runtime. Container instances are limited only by system resources, and the containers may consume resources at the expense of the Greenplum Database server. Future releases of Greenplum 7 may restore functionality to manage PL/Container resources using resource groups.
```
Comment (lisakowen) [inline gpdb-doc/markdown/analytics/pl_container_using.html.md:19]: do we also want this info in a Known Issues/ Limitations section?

File: gpdb-doc/markdown/analytics/pl_container_using.html.md [hunk 2]
```diff
@@ -5,105 +5,20 @@ title: Using PL/Container
...
-
-You cannot, by default, restrict the number of running PL/Container container instances, nor can you restrict the total amount of memory or CPU resources that they consume.
-
-### <a id="topic_resgroup"></a>Using Resource Groups to Manage PL/Container Resources 
-
-With PL/Container 1.2.0 and later, you can use Greenplum Database resource groups to manage and limit the total CPU and memory resources of containers in PL/Container runtimes. For more information about enabling, configuring, and using Greenplum Database resource groups, refer to [Using Resource Groups](../admin_guide/workload_mgmt_resgroups.html) in the *Greenplum Database Administrator Guide*.
-
-> **Note** If you do not explicitly configure resource groups for a PL/Container runtime, its container instances are limited only by system resources. The containers may consume resources at the expense of the Greenplum Database server.
```
Comment (lisakowen) [inline gpdb-doc/markdown/analytics/pl_container_using.html.md:22]: if we haven't, do we want to add the latter part of this note to the new section?

**Review by lisakowen — APPROVED** [review]

**Review by Sasasu — APPROVED** [review]
disabled in https://github.com/greenplum-db/plcontainer/pull/665

but will add it back soon

**Review by radarwave — APPROVED** [review]

File: gpdb-doc/markdown/analytics/pl_container_using.html.md (hunk 1 above)
Comment (dyozie) [reply gpdb-doc/markdown/analytics/pl_container_using.html.md:19]: Yes - this is already in place.

File: gpdb-doc/markdown/analytics/pl_container_using.html.md (hunk 2 above)
Comment (dyozie) [reply gpdb-doc/markdown/analytics/pl_container_using.html.md:22]: Yes - it's part of the new caution wording.

**Pull request merged**